.gitignore

```diff
@@ -1,9 +1,10 @@
-.sass-cache
-*.sublime-*
-local server.bat
-todo.txt
-build.js
-
-node_modules
-npm-debug.log*
-temp
+.sass-cache
+*.sublime-*
+local server.bat
+todo.txt
+build.js
+
+node_modules
+npm-debug.log*
+temp
+/.npmrc
```

Comment:
I have a global `.npmrc` config set for most of my projects; clobbered explicitly for KFC, but it's a good thing to ignore in general.